### DISCUSSION TITLE
"Direct / None" in Top Sources

### DISCUSSION THREAD

**Discussion by mchristofides** [plausible/analytics#814] [Anything Else]
When I'm trying to get a feeling for where our traffic is coming from, I find it really helpful to see the number of Direct visitors alongside the other sources. At the moment, I have to click on the "Details" of the "Top Sources" pain to see this.

Do others prefer it to be excluded in the default view, or could it go there too? I would personally find it helpful, mostly to get a quick understanding of the order of magnitude – e.g. is it most of what I'm looking at, or the minority? I find this tricky to tell at a glance from the Top Sources view at the moment. 

**Update:** "Direct / None" traffic is now shown in the main view of Top Sources as well!

**Comment by metmarkosaric** [comment]
thanks @mchristofides! when we were adding the "direct/none" source we asked the community about it and most people said that they're prefer it being away from the main view. most don't see it as actionable as other sources and also it stands for the majority of traffic for most sites which skews our little comparison we have on the dashboard.

**Reply by mchristofides** [reply]
@metmarkosaric thank you, good to know, I didn't realise it was the majority for most. Are there certain marketing strategies where Direct is more actionable, or at least more informative? Like if an SEO strategy is working well, would you start with a higher proportion of search engine traffic, but for the direct traffic to slowly grow as a proportion? Would be interested to hear if you would consider this as an option? I'm personally against having too many options in my own products, so totally understand if you're also trying to avoid this.

**Reply by metmarkosaric** [reply]
you're welcome! there's not much really you can do about direct traffic. and most of it is not necessarily people coming directly such as by typing the domain or clicking a bookmark. 

the only thing you can control is to UTM tag your links wherever you can such as emails, newsletters, messengers, social media, paid ads etc in order to minimize the number of clicks that go "dark" and show as direct. 

see https://plausible.io/blog/utm-tracking-tags

**Reply by mchristofides** [reply]
Thank you, I do understand that it isn't actionable in the same way as others, but can it be useful for understanding the business more widely? I think that's what I'm trying to do when looking for it.

Good tip re the UTM tags, I noticed you were doing that in a few places.

**Reply by metmarkosaric** [reply]
yeah, that's why we have it in "details". it's there for those that need it but it doesn't skew the comparison for those that don't need it

**Reply by mchristofides** [reply]
I was looking at your dashboard and realised that people using the hosted version of Plausible count as visitors, so makes sense that direct is so high.

I spotted that you include Direct when viewing “Realtime/Current Visitors” though, which I like. Are others more keen to see it there for some reason?

**Reply by metmarkosaric** [reply]
yeah, we decided to keep it on by default for the realtime view to showcase everything that's going on right now

**Reply by mmasouras** [reply]
We 'd like to see the Direct traffic included as well. The primary importance of the metric is that it communicates the level of our dependence to search and other traffic instead of users learning about our site and visiting it by themselves. I.e. it informs how many resources we allocate to SEO vs user outreach. 

Can we have a toggle in the Settings perhaps?

**Reply by metmarkosaric** [reply]
thanks @mmasouras! you can click on the "details" button in the Top Sources report to see Direct/None traffic too.

**Reply by mmasouras** [reply]
Yes, but I have to do this every time. Ideally, I 'd like to see that acquisition channel along with everything else. A setting would solve that problem, or just always show it.

**Comment by andrewholgate** [comment]
@mchristofides  There's the same discussion happening with others over on this ticket as well #1539

**Comment by FlorianWendelborn** [comment]
I too would like a toggle for this. Right now it’s not easy to tell if direct traffic is increasing or not. Since I’m working on a tool that I hope users will bookmark, direct traffic is a very important metric

**Reply by metmarkosaric** [reply]
thanks @FlorianWendelborn! you can click on direct/none to filter the dashboard by it and see the trend. see the example here on our live demo https://plausible.io/plausible.io?source=Direct+%2F+None&period=month

**Reply by FlorianWendelborn** [reply]
Ah that's very useful, thanks

**Comment by metmarkosaric** [comment]
**Update:** "Direct / None" traffic is now shown in the main view of Top Sources as well!

**Reply by soaresfellipe** [reply]
Hi @metmarkosaric. Will the "direct / none" traffic  be available on self-hosted versions too?

**Reply by metmarkosaric** [reply]
hi @soaresfellipe! self-hosted code is not updated as frequently as the Plausible managed hosting in the cloud, which changes daily. this means that the latest features won't be immediately available. we release a new self hosted version twice a year or so. so this will be live in the next version at some point later this year

**Reply by soaresfellipe** [reply]
Thanks, @metmarkosaric. :)
